go.mod

```diff
@@ -17,10 +17,10 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/tendermint/spm v0.0.0-20210524110815-6d7452d2dc4a
-	github.com/tendermint/tendermint v0.34.12
+	github.com/tendermint/tendermint v0.34.13
 	github.com/tendermint/tm-db v0.6.4
-	github.com/tharsis/ethermint v0.4.2-0.20210905110306-26c5eabb1893
-	google.golang.org/genproto v0.0.0-20210903162649-d08c68adba83
+	github.com/tharsis/ethermint v0.4.2-0.20210915094503-116de5461766
```

Comment:
the tag should be v0.5.0-x?

Reply:
It's figured out by `go mod tidy` automatically, not sure why it didn't pick the v0.5 tag